operations/conman/Troubleshoot_ConMan_Node_Pod_Stuck_Terminating.md

```diff
@@ -0,0 +1,99 @@
+# Troubleshoot Console Node Pod Stuck in Terminating State
+
+When a worker node goes down unexpectedly, a `cray-console-node` pod running on that worker
+may get stuck in a `Terminating` state that prevents it moving to a healthy worker. This can
+leave consoles unmonitored and not available for interactive access.
+
+## Prerequisite
+
+Determine if this is a scenario where the `cray-console-node` pod is stuck in `Terminating` due
+to the failure of the worker node it is running on or some other cause. This fix should only
+be applied to the case where the worker node did not properly drain before being shut down.
```

Comment:
```suggestion
Determine if this is a scenario where the `cray-console-node` pod is stuck in `Terminating` because
of the failure of the worker node it is running on, or for some other reason. This fix should only
be applied in the case where the worker node did not properly drain before being shut down.
```